cron.py

```diff
@@ -757,6 +761,72 @@ def delete_old_monitoring():
     LOG.d("delete monitoring records older than %s, nb row %s", max_time, nb_row)
 
 
+def check_hibp():
+    """
+    Check all aliases on the HIBP (Have I Been Pwned) API
+    """
+    LOG.d("Checking HIBP API for aliases in breaches")
+
+    if len(HIBP_API_KEYS) == 0:
+        LOG.exception("No HIBP API keys")
+        return
+
+    api_key_index = 0
+
+    LOG.d("Updating list of known breaches")
+    r = requests.get("https://haveibeenpwned.com/api/v3/breaches")
+    for entry in r.json():
+        hibp_entry = Hibp.get_by(name=entry['Name'])
+        if hibp_entry:
+            hibp_entry.description = entry['Description']
+        else:
+            hibp_entry = db.session.add(
+                Hibp(
```

Comment:
Nits: we can use `Hibp.create(...)` here as this method is provided by ModelMixin: https://github.com/simple-login/app/blob/30183ac8c3f705cac7d834dfe811b96b56ff81bc/app/models.py#L75

Reply:
I ended up using `get_or_create()`, as I only want to create it if it doesn't exist yet.